DigitalLearningSolutions.Data/DataServices/ConfigDataService.cs

```diff
@@ -58,6 +60,13 @@ public string GetConfigValueMissingExceptionMessage(string missingConfigValue)
         {
             return $"Encountered an error while trying to send an email: The value of {missingConfigValue} is null";
         }
+        public DateTime GetConfigLastUpdated(string key)

```

Comment:
I think there is an existing method to do this that was added when I added the Updated column to the config table.

Reply:
Looks like you have added `WHERE IsHtml = 1` to the clause in the existing query - why is this needed?

Reply:
I have remove the clause

Reply:
The new method is still there instead of making use of existing one. And I still see the clause being present there in the where condition. Please correct these two along with merge conflict.

Reply:
Please kindly check properly cause I have check again the last update shows that the clause is not part of the PR

Reply:
As explained over the call, please correct the above which is still pending.